config.js

```diff
@@ -0,0 +1,8 @@
+// All constants and configs go here
+const storageKey = 'covidStats';
+const bubbleCountKey = 'bubbleCount';
```

Comment:
Since we need to define const all the time we can define we can do something like this and export 
```
const constants = {
	covidStatsKey = 'covidStats',
	bubbleCountKey = 'bubbleCount'
}

export default constants
```

Reply:
Ok. when importing it back, shall we import the whole constant object or one by one? because, lets say a file uses only one constant, there is no need to import the entire object right? 
So, Whats better?
` import { covidStatsKey } from './constants.js `
or 
` import constants from './constants.js `

Reply:
Since we use only maximum 2,3 constants in one module I think we can use option 1 🙂 
It actually depends if object destruction is get bigger like more than 4 let's stick to option 2 🙂 
For now let's go with option 1

Reply:
So, if we are going with named option 1 then we have to do named exports. because object destructuring will not work for default export.

Reply:
Exactly, I forgot it, let's keep it as normal exports 🙂